application/templates/pages/about/performance.md

```diff
@@ -1,20 +1,20 @@
 Performance indicators show how the planning data platform is making it easier to find, use and trust planning and housing data in England.
 
-Last updated 31 July 2024.
+Last updated 28 August 2024.
 
 ## Availability
 
 We index planning and housing datasets provided by multiple organisations in England.
 
 Many datasets have one owner providing the data nationally. There are 8 datasets compiled from data sources provided by local planning authorities (LPAs) across England.
 
-### 89 datasets on the platform
+### 93 datasets on the platform
 
-89 datasets available from 452 organisations. <!--? added last month (+/-0%). -->
+93 datasets available from 377 organisations. <!--? added last month (+/-0%). -->
```

Comment:
Where does the 377 figure come from?

Reply:
OK, great, thank you! Original figure was a mistake, but good to have this recorded.